systers_portal/templates/meetup/snippets/meetup_locations_grid.html

```diff
@@ -0,0 +1,18 @@
+<div class="row box-body">
+  <div class="col-xs-offset-1 col-xs-5">
```

Comment:
And for the rows we can do the following: remove the offset, just use the columns. Set a half for a each div with `col-md-6` for medium devices and for small ones take the full width using `col-xs-12`. So on big screens we will get 2 columns, but on small ones just 1. 

Everything else looks very good :+1: